messaging/src/main/java/org/axonframework/commandhandling/gateway/IntervalRetryScheduler.java

```diff
@@ -134,8 +136,8 @@ private boolean scheduleRetry(Runnable commandDispatch, int interval) {
      */
```

Comment:
The Builder class level javadoc should state clearly what the retry interval and max retry count are defaulted too.

Reply:
This requirement also holds for the `builder()` function.